tests/topotests/ospf_basic_functionality/test_ospf_ecmp.py

```diff
@@ -216,6 +216,19 @@ def red_connected(dut, config=True):
     assert result is True, "Testcase: Failed \n Error: {}".format(result)
 
 
+def redistribute(dut, route_type, **kwargs):
```

Comment:
do we really need 6 different copies of the same function?  Shouldn't this belong in lib/ospf.py?

Reply:
Indeed, but as they originally were implemented in each test I thought of keeping like that. 
As the function is part of the lib now, I renamed it `redistribute_ospf`